docs/develop/node/archival/hardware-archival.md

```diff
@@ -53,3 +53,6 @@ All prices reflect *reserved instances* which offer deep discounts on all platfo
 - Azure — https://azure.microsoft.com/en-us/pricing/calculator
 
 </blockquote>
+
+> Got a question?
+> <a href="https://stackoverflow.com/questions/tagged/nearprotocol"> > <h8>Ask it on StackOverflow!</h8></a>
```

Comment:
Weirdly, this renders extra `>` in the text after "Got a question?"

https://docs-pr-742.onrender.com/docs/develop/node/archival/hardware-archival

![image](https://user-images.githubusercontent.com/304265/121682027-ebcbc200-ca91-11eb-881c-24f2f533dfd4.png)